about.html

```diff
@@ -91,8 +91,13 @@ <h3>CREATIVE DESIGN<br>GLOBAL SUMMIT 2023 Logo</h3>
         <h3><b>See the award CD Global Summits<br><span>2023</span></b></h3>
         <div class="line" id="lineFeature"></div>
         <p>The award for this first convention Create Design Tattoo is a new Kit SnoopDragon Edicion SnoopDog.</p>
-        <div class="sectionFeature" id="prize1">
-          <img src="./img/Icons/snoopDragon1.png" alt="Award for first place">
+        <div class="containerImg">
+          <div class="sectionFeature" id="prize2">
+            <img src="./img/Icons/Template.png" alt="Prize for the first place">
+          </div>
+          <div class="sectionFeature" id="prize1">
+            <img src="./img/Icons/snoopDragon1.png" alt="Award for first place">
+          </div>
         </div>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure the above section sticks as much as possible to the template design. The below images should have a background color 

### Template design
![Screenshot from 2023-02-18 23-07-33](https://user-images.githubusercontent.com/83042742/219899557-4357eee7-83f8-47c4-849e-69a09af2fb57.png)

### Your design
![Screenshot from 2023-02-18 23-13-45](https://user-images.githubusercontent.com/83042742/219899838-30293ad7-625e-444b-bfa8-ef878c770f90.png)

Reply:
> Correction 

Add a background and stick to the template design

![imagen](https://user-images.githubusercontent.com/90431831/219906064-78871138-60b2-4590-99ec-c3b1776845db.png)